mapper-sdk-go/internal/mqttadapter/subscribe.go

```diff
@@ -43,10 +43,6 @@ func SyncInfo(dic *di.Container, message mqtt.Message) {
 		if i == len(deviceInstances[instanceID].Twins) {
 			continue
 		}
-		// Desired value is not changed.
-		if deviceInstances[instanceID].Twins[i].Desired.Value == twinValue {
```

Comment:
why delete this line? 
if the desired value is not changed, we should set the value to device.

Reply:
The logic processing in this place is not suitable for control message processing, such as controlling the camera to take pictures, one shot is fine, but two shots are not.
Currently, kubeedge does not support independent control messages, so the limitation of this place is removed first.

Reply:
If removed, whether it will affect the original processing logic? Can we add some new logic to handle the situation you mentioned?

Reply:
If removed, whether it will affect the original processing logic? Can we add some new logic to handle the situation you mentioned?